gfx.cpp

```diff
@@ -1561,20 +1561,16 @@ class GfxInternal
             return buffer;
         }
         if(cpu_access != kGfxCpuAccess_None)
-            switch(cpu_access)
+            if(cpu_access == kGfxCpuAccess_Write)
             {
-            case kGfxCpuAccess_Write:
-                {
-                    D3D12_RANGE read_range = {};
-                    gfx_buffer.resource_->Map(0, &read_range, &gfx_buffer.data_);
-                    memset(gfx_buffer.data_, 0, (size_t)resource_desc.Width);
-                    if(data) memcpy(gfx_buffer.data_, data, (size_t)size);
-                }
-                break;
-            default:
-                gfx_buffer.resource_->Map(0, nullptr, &gfx_buffer.data_);
-                break;
+                D3D12_RANGE read_range = {};
+                gfx_buffer.resource_->Map(0, &read_range, &gfx_buffer.data_);
+                memset(gfx_buffer.data_, 0, (size_t)resource_desc.Width);
+                if(data)
+                    memcpy(gfx_buffer.data_, data, (size_t)size);
             }
+            else
+                gfx_buffer.resource_->Map(0, nullptr, &gfx_buffer.data_);
```

Comment:
switch statements with only 1 case are apparently frowned upon. So converted to simple if/else